clusterctl-settings.json

```diff
@@ -0,0 +1,7 @@
+{
+    "name": "infrastructure-tinkerbell",
```

Comment:
Do we want it `infrastructure-tinkerbell` or `infrastructure-tink` ? #11

Reply:
Oh, I see next commits now.

Reply:
I think we want tinkerbell here, since this will be used by clusterctl for output.